sourcegraph/about
# Gated Pg/ Gartner report

This closes #5369. It adds a new gated report page using our new simple gated page layout

### Changelog
- Adds `/2022-gartner-cool-vendors-in-software-engineering`

### Note
- This merges into the open Guide/vulernabilities PR to illustrate the limited updates. Once that PR is merged, GH automatically changes the merge destination to: `main`

### Test
1. Navigate to `/2022-gartner-cool-vendors-in-software-engineering`
2. Please check that layout matches Figma specs
3. Ensure linting passes.
4. Ensure prettier has standardized the proposed changes.
5. Ensure dev and production builds work.
